app/models/group.rb

```diff
@@ -506,9 +506,20 @@ def usernames
 
   PUBLISH_CATEGORIES_LIMIT = 10
 
-  def add(user)
+  def add(user, opts = {})
```

Comment:
I would not rush to adding an `opts` here and just add a new `notify` parameter with a default value.

```suggestion
  def add(user, notify: false)
```